packages/ember-application/tests/system/dependency_injection/default_resolver_test.js

```diff
@@ -126,7 +126,7 @@ QUnit.test('the default resolver resolves helpers on the namespace', function()
 
   expectDeprecation(function() {
     LegacyHTMLBarsBoundHelper = makeHTMLBarsBoundHelper(function() {});
-  }, 'Using `Ember.HTMLBars.makeBoundHelper` is deprecated. Please refactor to using `Ember.Helper` or `Ember.Helper.helper`.');
```

Comment:
This is testing that a specific deprecation message is issued, if you change the expected message you also need to change where it is triggered.